### PR TITLE
[ Xedra Evolved ] Paraclesian's shouldn't have drivers license

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
Mods "Prevent Paraclesians from being born with a drivers license"

#### Purpose of change
Limits Paraclesian hobbies to the Paraclesian birth hobby granting deduction.

#### Describe the solution
Use the hobbies field in professions.

#### Describe alternatives you've considered
None

#### Testing
Seems to be working

### CODE REVIEW

**Comment by Maleclypse** [comment]
Note to self: Hommullus should have access to Birthmark hobbies that block other birthmarks but grant a special ability of minor usage. The Key, should unlock a door once a day.

File: data/mods/Xedra_Evolved/player/hobbies.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    "id": "SYLPH_EVERYWHERE_NOWHERE",
+    "name": "Everywhere and nowhere.",
+    "description": "Air expands to fill every space it encounters and by filling the space it becomes diffuse and impossible to grasp.  Would you slip through the fingers of those who grasp at you?",
+    "points": 3,
+    "skills": [ { "level": 3, "name": "dodge" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "ARVORE_ROOT_AND_SOIL",
```
Comment (Fris0uman) [inline data/mods/Xedra_Evolved/player/hobbies.json:41]: Giving the same ID to the trait and the profession might lead to confusion

**Review by Fris0uman — COMMENTED** [review]
Tests are failing on missing traits

File: data/mods/Xedra_Evolved/mutations/paraclesians/arvore_mutations.json [hunk 2]
```diff
@@ -129,6 +129,18 @@
     "spells_learned": [ [ "arvore_nurture_the_green", 1 ] ],
     "//": "This only causes growth in existing plants.  Growing new plants from nothing should be advanced mutations/spells."
   },
+  {
+    "type": "mutation",
+    "id": "ARVORE_ROOT_AND_SOIL",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/mutations/paraclesians/arvore_mutations.json:134]:
```suggestion
    "id": "ARVORE_NURTURE_GREEN",
```

File: data/mods/Xedra_Evolved/player/hobbies.json (hunk 1 above)
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/player/hobbies.json:41]:
```suggestion
    "id": "ARVORE_NURTURE_GREEN",
```

File: data/mods/Xedra_Evolved/player/hobbies.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    "name": "The way of root and soil.",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/player/hobbies.json:42]:
```suggestion
    "name": "Born a seedbearer.",
```

File: data/mods/Xedra_Evolved/mutations/paraclesians/arvore_mutations.json [hunk 4]
```diff
@@ -129,6 +129,18 @@
     "spells_learned": [ [ "arvore_nurture_the_green", 1 ] ],
     "//": "This only causes growth in existing plants.  Growing new plants from nothing should be advanced mutations/spells."
   },
+  {
+    "type": "mutation",
+    "id": "ARVORE_NURTURE_GREEN",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/mutations/paraclesians/arvore_mutations.json:134]:
```suggestion
    "id": "ARVORE_SEED_BEARER",
```

File: data/mods/Xedra_Evolved/player/professions.json [hunk 5]
```diff
@@ -528,6 +529,7 @@
     "traits": [ "ARVORE" ],
     "pets": [ { "name": "mon_vineling", "amount": 1 } ],
     "recipes": [ "mutagen_plantkin", "verdant_poultice" ],
+    "hobbies": [ "PARACLESIAN_BIRTH", "ARVORE_ROOT_AND_SOIL" ],
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/player/professions.json:532]:
```suggestion
    "hobbies": [ "PARACLESIAN_BIRTH", "ARVORE_NURTURE_GREEN" ],
```

File: data/mods/Xedra_Evolved/mutations/paraclesians/ierde_eocs.json [hunk 6]
```diff
@@ -49,6 +49,24 @@
...
+        },
+        {
+          "condition": { "math": [ "u_val('morale')", "<", "50" ] },
+          "msg_on": { "text": "Everything is wonderful and this too shall pass.", "rating": "mixed" }
+        }
+      ]
+    },
+    "effect": [ { "math": [ "u_val('morale')", "=", "0" ] } ],
+    "false_effect": [ { "u_message": "You are already in the mindset of stone.  Patient and eternal.", "type": "mixed" } ]
+  },
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/mutations/paraclesians/ierde_eocs.json:69]:
```suggestion
something like this is accepted when creating a new game
```
  {
    "id": "EOC_IERDE_STONE_MEDITATIONS",
    "type": "effect_on_condition",
    "condition": {
      "or": [
        { "condition": { "math": [ "u_val('morale')", "==", "0" ] } },
        { "condition": { "math": [ "u_val('morale')", "<", "-50" ] } },
        { "condition": { "math": [ "u_val('morale')", ">", "50" ] } }
      ]
    },
    "effect": [ { "math": [ "u_val('morale')", "=", "0" ] } ],
    "false_effect": [ { "u_message": "You are already in the mindset of stone.  Patient and eternal.", "type": "mixed" } ]
  },
```
```

File: data/mods/Xedra_Evolved/mutations/paraclesians/ierde_eocs.json [hunk 7]
```diff
@@ -49,6 +49,7 @@
       }
     ]
   },
+something like this is accepted when creating a new game
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/mutations/paraclesians/ierde_eocs.json:52]:
```suggestion
```

**Comment by Maleclypse** [comment]
Need to add the spell still.

File: data/mods/Xedra_Evolved/mutations/paraclesians/arvore_mutations.json [hunk 8]
```diff
@@ -129,6 +129,18 @@
     "spells_learned": [ [ "arvore_nurture_the_green", 1 ] ],
     "//": "This only causes growth in existing plants.  Growing new plants from nothing should be advanced mutations/spells."
   },
+  {
+    "type": "mutation",
+    "id": "ARVORE_SEED_BEARER",
+    "name": { "str": "Nurturing the Green" },
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/mutations/paraclesians/arvore_mutations.json:135]:
```suggestion
    "name": { "str": "Seed Bearer" },
```

**Pull request merged**